src/layouts/app/pages/NoPetRegirested/index.js

```diff
@@ -30,6 +31,11 @@ export default function NoPetRegirested() {
     .get('button')
     .classList.add('no-pet-regirested-page__button');
   this.button.mount($container);
+
+  const $buttonElement = this.button.selected.get('button');
+  $buttonElement.addEventListener('click', () => {
+    Router.go('/app/addpets');
+  });
```

Comment:
por que nao usar o evento do component?
```suggestion
  this.button.listen('click', () => {
    Router.go('/app/addpets');
  });
```